### DISCUSSION TITLE
Want to "release" subdomain, no longer need it/never finished web page

### DISCUSSION THREAD

**Discussion by gingerchew** [is-a-dev/register#2490] [Q&A] [answered]
I had an idea for this on the back-burner for forever, but never got around to it. Since then I've changed my username/online identity, and interest in the idea has waned. I was wondering what I needed to do to "release" it for other people if they wanted it. 

Should I just make a PR with that file removed? Or is there something special that needs to happen on y'alls end? 

Thanks :)

**Comment by phenax** [comment, marked as answer]
You can create another pull request deleting the json file. Also, we appretiate you wanting to free up the un-used subdomain for someone else!

**Reply by gingerchew** [reply]
Well it just seemed like the responsible thing to do :) Opened the PR at #2510
